src/mail/ngx_mail_core_module.c

```diff
@@ -468,6 +468,11 @@ ngx_mail_core_listen(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
 #endif
         }
 
+        if (ngx_strcmp(value[i].data, "mptcp") == 0) {
```

Comment:
out of curiosity: why did you put this block here and not at the end? Same in the other modules. I don't know if there is an order (alphabetical, when they are added, etc.)

Reply:
I very well could have, but it seemed more readable to me since the next parameter (so_keepalive) is quite long.
If it is a bother, it can be moved.